specification/columns/skupriceid.md

```diff
@@ -2,7 +2,7 @@
 
 A SKU Price ID is a unique identifier that defines the unit price used to calculate the charge. SKU Price ID can be referenced on a price list published by a provider to look up detailed information, including a corresponding list unit price. The composition of the properties associated with the SKU Price ID may differ across providers. SKU Price ID is commonly used for analyzing cost based on pricing properties such as Terms and Tiers.
 
-The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The ListUnitPrice MUST be associated with the SKUPriceID in the provider published price list. The SkuPriceId MUST NOT be null when ChargeType is "Purchase" or "Usage". SkuPriceId MUST NOT be null when ChargeSubcategory is "Refund" and the refund is related to charges with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one SkuId.
+The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SKUPriceID in the provider published price list. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage". SkuPriceId MUST NOT be null when [ChargeSubcategory](#chargesubcategory) is "Refund" and the refund is related to charges with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid).
```

Comment:
I have an clarification to add here.  I discovered that a SKU Price can be related to multiple SKUs as it pertains to Commitment Discount flexibility (see [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/apply_ri.html) for AWS, [here](https://learn.microsoft.com/en-us/azure/virtual-machines/reserved-vm-instance-size-flexibility) for Azure).  If the scope here is too large for a final scrub, lmk and we can discuss further and/or we can add more context, either here or in the glossary; if FOCUS contributors have defined a provider-agnostic term for this CD flexibility concept, I haven't heard it.

```suggestion
The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SKUPriceID in the provider published price list. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage". SkuPriceId MUST NOT be null when [ChargeSubcategory](#chargesubcategory) is "Refund" and the refund is related to charges with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.
```